### PR TITLE
Log out during a skipped spec

### LINKED ISSUE #166: Focused tests should be called out in the console log
image

I have about 80 tests that I've written in Specta and have for a while been regularly using with no issue, but just now as I was developing I must have changed something because now none of my tests will run.  The console output reports that each test has run (mentioning each individually), and if I can hit breakpoints in the `it` and `describe` statements and see that they are getting run.  However, none of the code in each of the test blocks appears to be executing.  What did I do?  I'm running Specta 0.5.

image

### PR DESCRIPTION
**Pull request by orta**
Fixes #166 - wasn't really too sure how to test this, I don't want to try and do anything funny with NSLog swizzling, so I guess this really is all there needs to be.

### CODE REVIEW

**Comment by robb** [comment]
I take it this will have the name of the spec directly before it?

File: Specta/Specta/SPTSpec.m [hunk 1]
```diff
@@ -145,6 +145,7 @@ - (void)spt_runExample:(SPTCompiledExample *)example {
     example.block(self);
   } else if (!example.pending) {
     self.spt_skipped = YES;
+    NSLog(@"Skipping Test");
```
Comment (modocache) [inline Specta/Specta/SPTSpec.m]: I believe the `example` has a property for its name--why not log that as well, so users know which tests are being skipped?

**Comment by orta** [comment]
screen shot 2016-02-11 at 15 25 36

Happy to add the example name, it is both on the line above and the line below too

**Comment by modocache** [comment]
@robb Oh, good point.

Let's merge this sucker.

**Comment by orta** [comment]
:+1:

**Comment by robb** [comment]
works for me

**Pull request merged**